api/go.mod

```diff
@@ -4,6 +4,7 @@ go 1.13
 
 require (
 	github.com/Microsoft/go-winio v0.4.14
```

Comment:
fixed

Reply:
seems like CI is failing because of this. I'm trying to figure out how to handle binary dependencies

Reply:
There are ways to combine modules and tools, i.e. make tools to survive `go mod tidy`. I don't have battle scars with any of them yet, my personal preference would be to manage such tools outside of repo. 
In our case, I think a solid approach might be to configure CI pipeline with a new binary to publish documentation. That would make sure that tools' code won't be compiled into the portainer binary.
Alternatively, I can think of using build flags. We can configure pipeline to do a separate build including tools and use it for documentation, while a different build will be packaged for distribution.

Reply:
For now, I've remove the go.mod requirement and added a `prebuild:docs` script to package,json. this will take care of installing swagger anytime running `yarn build:docs`